analysis/org.eclipse.tracecompass.analysis.counters.core/src/org/eclipse/tracecompass/analysis/counters/core/CounterStateProvider.java

```diff
@@ -147,8 +146,9 @@ protected void eventHandle(@NonNull ITmfEvent event) {
      *            Grouped counter aspect
      * @param rootQuark
      *            Key to a relative root of the state system
+     * @since 3.0
      */
-    protected void handleGroupedCounterAspect(ITmfEvent event, ITmfStateSystemBuilder ss, CounterAspect aspect, int rootQuark) {
+    protected void handleGroupedCounterAspect(ITmfEvent event, ITmfStateSystemBuilder ss, ITmfCounterAspect aspect, int rootQuark) {
```

Comment:
This is detected as major API change. I think you have to keep the old signature and deprecated it, and have it call the new signature, casting the aspect to avoid recursion.

With this change, it will only require a minor version bump to 2.3.0, this must be done in the plug-in MANIFEST.MF and will affect all updated @since tags in this commit, to be set to 2.3.